doc/UltiSnips.txt

```diff
@@ -302,7 +303,33 @@ then you define your mapping as >
 and if the you can't expand or jump from the current location then the
 alternative function IMAP_Jumpfunc('', 0) is called.
 
-3.2.2 Path to Python module                   *UltiSnips-python-module-path*
+3.2.2 Inner mappings                          *UltiSnips-inner-mappings*
+--------------------
+
+                                *UltiSnipsMapInnerKeys* *UltiSnipsUnmapInnerKeys*
+For maximum compatibility with other plug-ins, UltiSnips sets up some special
+mappings when a snippet starts being expanded, and tears them down once
+expansion is completed. In order to make it possible to override these inner
+mappings, it fires the following "User" autocommands:
+
+UltiSnipsMapInnerKeys
+UltiSnipsUnmapInnerKeys
+
+For example, to call a pair of custom functions in response to these events,
+you might do: >
+
+   autocmd! User UltiSnipsMapInnerKeys
+   autocmd User UltiSnipsMapInnerKeys call CustomInnerKeyMapper()
+   autocmd! User UltiSnipsUnmapInnerKeys
+   autocmd User UltiSnipsUnmapInnerKeys call CustomInnerKeyUnmapper()
+
+Note that because snippet expansion may be nested, it is possible that
```

Comment:
I think this is a unsymmetry that can be avoided with my suggested names up there.